src/AD_ForwardDiff.jl

```diff
@@ -4,6 +4,8 @@
 
 import .ForwardDiff
```

Comment:
I already assumed that Requires chokes with `using ForwardDiff`/`import ForwardDiff`. I guess the following works:
```suggestion
import .ForwardDiff: ForwardDiff, DiffResults
```